gonomics/bedtoolsbedopsIntervalOverlap/bedtoolsbedopsIntervalOverlap20230707_test.go

```diff
@@ -0,0 +1,1836 @@
+	gonomicsSingleThread1mi_10k = "csh/gonomicsSingleThread/a1mi/intervalOverlapSingleThread1mi_10k.csh"
+	gonomicsSingleThread1mi_100k = "csh/gonomicsSingleThread/a1mi/intervalOverlapSingleThread1mi_100k.csh"
+	gonomicsSingleThread1mi_1mi = "csh/gonomicsSingleThread/a1mi/intervalOverlapSingleThread1mi_1mi.csh"
+	gonomicsSingleThread1mi_10mi = "csh/gonomicsSingleThread/a1mi/intervalOverlapSingleThread1mi_10mi.csh"
+	// a10mi
+	gonomicsSingleThread10mi_100 = "csh/gonomicsSingleThread/a10mi/intervalOverlapSingleThread10mi_100.csh"
+	gonomicsSingleThread10mi_1k = "csh/gonomicsSingleThread/a10mi/intervalOverlapSingleThread10mi_1k.csh"
+	gonomicsSingleThread10mi_10k = "csh/gonomicsSingleThread/a10mi/intervalOverlapSingleThread10mi_10k.csh"
+	gonomicsSingleThread10mi_100k = "csh/gonomicsSingleThread/a10mi/intervalOverlapSingleThread10mi_100k.csh"
+	gonomicsSingleThread10mi_1mi = "csh/gonomicsSingleThread/a10mi/intervalOverlapSingleThread10mi_1mi.csh"
+	gonomicsSingleThread10mi_10mi = "csh/gonomicsSingleThread/a10mi/intervalOverlapSingleThread10mi_10mi.csh"
+
+	// gonomicsA1kIntermediateThread
+	gonomicsA1kIntermediateThread1k_100 = "csh/gonomicsA1kIntermediateThread/a1k/intervalOverlapGonomics1k_100.csh"
+	gonomicsA1kIntermediateThread1k_1k = "csh/gonomicsA1kIntermediateThread/a1k/intervalOverlapGonomics1k_1k.csh"
+	gonomicsA1kIntermediateThread1k_10k = "csh/gonomicsA1kIntermediateThread/a1k/intervalOverlapGonomics1k_10k.csh"
+	gonomicsA1kIntermediateThread1k_100k = "csh/gonomicsA1kIntermediateThread/a1k/intervalOverlapGonomics1k_100k.csh"
+	gonomicsA1kIntermediateThread1k_1mi = "csh/gonomicsA1kIntermediateThread/a1k/intervalOverlapGonomics1k_1mi.csh"
+	gonomicsA1kIntermediateThread1k_10mi = "csh/gonomicsA1kIntermediateThread/a1k/intervalOverlapGonomics1k_10mi.csh"
+
+	// gonomicsA1kOptimalThread
+	gonomicsA1kOptimalThread1k_100 = "csh/gonomicsA1kOptimalThread/a1k/intervalOverlapGonomics1k_100.csh"
+	gonomicsA1kOptimalThread1k_1k = "csh/gonomicsA1kOptimalThread/a1k/intervalOverlapGonomics1k_1k.csh"
+	gonomicsA1kOptimalThread1k_10k = "csh/gonomicsA1kOptimalThread/a1k/intervalOverlapGonomics1k_10k.csh"
+	gonomicsA1kOptimalThread1k_100k = "csh/gonomicsA1kOptimalThread/a1k/intervalOverlapGonomics1k_100k.csh"
+	gonomicsA1kOptimalThread1k_1mi = "csh/gonomicsA1kOptimalThread/a1k/intervalOverlapGonomics1k_1mi.csh"
+	gonomicsA1kOptimalThread1k_10mi = "csh/gonomicsA1kOptimalThread/a1k/intervalOverlapGonomics1k_10mi.csh"
+)
+
+// for benchmarking memory usage
+var memUsage []float64
+var cmd *exec.Cmd
+
+// benchmarking calls
+
+// gonomicsA1kOptimalThread
+func BenchmarkGonomicsA1kOptimalThreadIntervalOverlap1k_100(b *testing.B) {
+	for i := 0; i < b.N; i++ {
+		cmd = gonomicsIntervalOverlap(gonomicsA1kOptimalThread1k_100)
+		b.StopTimer() // prevent memUsage append from counting in benchmarking time
+		memUsage = append(memUsage, float64(cmd.ProcessState.SysUsage().(*syscall.Rusage).Maxrss))
```

Comment:
I had thought we changed and were not doing mem usage this way for intervalOverlap?

Reply:
Just removed all mem usage code